src/debug/ee/arm64/arm64walker.cpp

```diff
@@ -178,17 +177,72 @@ BYTE*  NativeWalker::SetupOrSimulateInstructionForPatchSkip(T_CONTEXT * context,
     else if(RegNum >= 0)
     {
         CORDbgSetInstruction((CORDB_ADDRESS_TYPE *)patchBypass, 0xd503201f); //Add Nop in buffer
+
         PCODE RegContents;
-        if ((opcode & 0xFF000000) == 0x58000000) //Actual LDR literal
+        if ((opcode & 0x3BF000000) == 0x18000000) //LDR Literal
```

Comment:
This should be:

 if ((opcode & 0x3B0000000) == 0x18000000) //LDR Literal (General or SIMD)